src/import/sst_importer.rs

```diff
@@ -0,0 +1,432 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
```

Comment:
Could you use `util::collections::HashMap` ?

Reply:
I think it doesn't have any performance problem here, so I prefer a more common data structure here.

Reply:
we only use standard HashMap in the test, so I prefer keeping the same with other mods.